### PR TITLE
Credmark

### PR DESCRIPTION
**Pull request by murphy**
Adding the Credmark project.

### CODE REVIEW

File: protocols/credmark/index.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "cname": "Credmark",
```
Comment (CarlGocht) [inline protocols/credmark/index.json:2]: can you make this all lowercase?

Comment (murphy) [reply protocols/credmark/index.json]: Done & pushed

**Comment by Rashmi-278** [comment]
@murphy  Can you drop a link to your DAO ? It would help us integrate sooner.

**Comment by murphy** [comment]
Someone asked for protocol info. Our protocol isn't live yet.

**Review by CarlGocht — DISMISSED** [review]

File: protocols/credmark/index.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-    "cname": "Credmark",
+    "cname": "credmark",
     "description": "High integrity DeFi data and a risk modeling platform",
     "path": "credmark",
-    "isEnabled": false
+    "isEnabled": false,
```
Comment (Rashmi-278) [inline protocols/credmark/index.json:5]: @murphy  Can you please remove the comma after the value 'false' ?

Comment (Rashmi-278) [reply protocols/credmark/index.json:5]: or maybe add 
`"type": "snapshot"`
after that

Comment (murphy) [reply protocols/credmark/index.json:5]: Added "type": "snapshot".

🤞🏻 that gets us over the line.

**Review by Rashmi-278 — APPROVED** [review]

**Comment by sudheerDev** [comment]
@murphy Credmark is live now 🎉 , we had another PR for it that got merged

**Pull request closed without merging**